aspnetcore/razor-pages/rp-mvc.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Razor Pages compared to ASP.NET Core MVC with controllers and views
+author: Rick-Anderson
+description: Contrast and compare Razor Pages to controller/view programming in ASP.NET Core
+monikerRange: '>= aspnetcore-2.0'
+ms.author: riande
+ms.date: 8/05/2018
+uid: razor-pages/rp-mvc
+---
+# Razor Pages compared to ASP.NET Core MVC
+
+By [Scott Sauber](https://twitter.com/scottsauber) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Razor Pages is a feature of the ASP.NET Core MVC framework. Razor Pages follows the MVVM pattern. Developers and teams familiar with ASP.NET MVC development:
+
+* Can continue app development featuring controllers and views.
+* Can be assured the ASP.NET Core framework will provide improvements to controller/view development.
+
+Razor Pages can be added to controller/view projects. The controller-view approach to MVC may be preferred by larger teams.
+
+If you haven't tried Razor Pages, consider using them. Many developers find Razor Pages development simpler and more productive than using controllers and views.
+
+Razor Pages advantages over controller/view development:
```

Comment:
IF we're going to present differences as advantages for RP, we should be sure to include differences that are advantages for MVC as well.

Reply:
Another thought on this comment occurs to me -- if we think MVC has no advantages, and if this article is really not so much about comparing the two as it is about "why RP is better than controllers/views and why you should try it", I would consider incorporating this content into the RP intro instead of this separate doc.